pkg/mappings.bzl

```diff
@@ -92,6 +98,8 @@ def pkg_attributes(mode = None, user = None, group = None, **kwargs):
       mode: string: UNIXy octal permissions, as a string.
       user: string: Filesystem owning user.
       group: string: Filesystem owning group.
+      uid: string: Filesystem owning uid.
+      gid: string: Filesystem owning gid.
```

Comment:
Perhaps expand the acronyms here?  e.g. uid = "user id".

Reply:
Done.